### PR TITLE
Updating AdSense Python examples for v2.

### PR DESCRIPTION
**Pull request by donovanfm**
Also update top-level README about the example rewrite. (Other languages to come in future pull requests.)

### CODE REVIEW

File: v2/python/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# AdSense Management API v1.4 Samples
```
Comment (abadreldin) [inline v2/python/README.md:1]:
```suggestion
# AdSense Management API v2 Samples
```

File: v2/python/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - https://github.com/googleapis/google-auth-library-python-oauthlib
+
+  or use pip:
+
+  ```bash
+  $ pip install google-api-python-client
+  $ pip install google_auth_oauthlib
+  ```
+
+2. (No longer needed, check on this) Make sure you can import the client library:
```
Comment (abadreldin) [inline v2/python/README.md:20]: Just follow up on if this step is needed.

File: v2/python/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  ```
+  $ python
+  >>> import apiclient
+  ```
+
+3. Execute any of the scripts to begin the auth flow:
+
+  ```bash
+  $ python get_all_accounts.py
```
Comment (abadreldin) [inline v2/python/README.md:30]:
```suggestion
  $ python3 get_all_accounts.py
```

File: v2/python/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  ```
+
+  A browser window will open and ask you to login. Use the AdSense account.
+
+  Note: some examples require an argument to be passed in in order to run,
```
Comment (abadreldin) [inline v2/python/README.md:35]: Suggestion to add a note that "To determine what arguments are required you can pass in `help` as an argument, e.g., `python3 get_all_ad_units.py help`".

File: v2/python/README.md (hunk 2 above)
Comment (donovanfm) [reply v2/python/README.md]: Ah yes, the lingering TODO. Actually, this step still makes sense. I updated the name of the client library in my latest commit.

File: v2/python/README.md (hunk 3 above)
Comment (donovanfm) [reply v2/python/README.md:30]: I actually think most people have mapped `python` to version 3.x at this point. Python 2 has been sunset for over a year. If you're fine with leaving this as is, can you resolve this conversation?

**Review by abadreldin — APPROVED** [review]
LGTM

**Pull request merged**